coalib/collecting/Collectors.py

```diff
@@ -194,6 +194,35 @@ def filter_section_bears_by_languages(bears, languages):
     return new_bears
 
 
+def filter_capabilities_by_languages(bears, languages):
+    """
+    Filters the bears capabilities by languages.
+
+    :param bears:       the dictionary of the sections as keys and list of
+                        bears as values.
+    :param languages:   languages that bears are being filtered on.
+    :return:            new dictionary with languages as keys and their bears
+                        capabilities as values. The capabilities are stored in a
+                        tuple of two element where the first one represent what
+                        the bears can detect, and the second one what they can
+                        fix.
+    """
+    languages = set(language.lower() for language in languages) | {'all'}
+    language_bears_capabilities = {language: (
+        set(), set()) for language in languages}
+    for section in bears.keys():
```

Comment:
rather do `for section_bears in bears.values()` so you don't have to do `bears[section]` later, just iterate over values directly?